spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -928,10 +929,59 @@
         id_token_hint: id_token,
       )
 
+      expected_redirect_uri = URI('gov.gsa.openidconnect.test://result/signout').
+        tap { |uri| uri.query = URI.encode_www_form(state:) }.
+        to_s
```

Comment:
we could use our `UriService.add_params` here if we want
```suggestion
      expected_redirect_uri = UriService.add_params('gov.gsa.openidconnect.test://result/signout', state:)
```

Reply:
> we could use our `UriService.add_params` here if we want

Yep, good suggestion. I'd even went looking for a Ruby utility to help here, didn't think to check our own utilities 😅

Reply:
Updated in 9f8b07c.